dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -56,6 +55,8 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
             hr = channelCloseException.code();
 
+            logTelemetry.StopWithResult(hr);
```

Comment:
Antipattern. No need of stop. The throw below in the if condition should log the hresult automatically.

Reply:
ERROR_NOT_FOUND is by design if the registration doesn't exist. So its ok to ignore. Please remove the stop event.

Reply:
Actually ERROR_NOT_FOUND is still being logged because of the THROW macro above. I'm going to remove the StopWithResult().